### PR TITLE
Avatar,badge

### PR DESCRIPTION
**Pull request by NeerajVodala**
Done with the avatar and badge components. Please review, Thanks.

### CODE REVIEW

File: docs/components/badge.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+            <a href="https://github.com/NeerajVodala/component-library" target="_blank"><i
+                    class="fab fa-github fa-lg"></i></a>
+
+            <a href="#" target="_blank"><i class="fab fa-discord fa-lg"></i></a>
+
+            <a href="https://twitter.com/NeerajVodala21" target="_blank"><i class="fab fa-twitter fa-lg"></i></a>
+
+            <a href="#"><i class="fas fa-sun fa-lg"></i></a>
+
+            <!-- <a href="#"><i class="fas fa-moon fa-lg"></i></a> -->
```
Comment (AtulHande03a) [inline docs/components/badge.html]: remove these comments

File: styles/cluach.css [hunk 2]
```diff
@@ -61,17 +61,17 @@
 /* dp - docs page */
 .dp {
     display: flex;
-    margin-top: 2rem;
+    margin: 2rem 0rem;
```
Comment (AtulHande03a) [inline styles/cluach.css]: 0 is fine no need for 0 rem

**Review by AtulHande03a — APPROVED** [review]
Apart from the mentioned minor comments , rest looks good!!

File: docs/components/badge.html (hunk 1 above)
Comment (NeerajVodala) [reply docs/components/badge.html]: done

**Review by PrashanthreddyAinala — COMMENTED** [review]
Looks good for me

**Pull request merged**